lib/inferno/dsl/jwks.rb

```diff
@@ -0,0 +1,27 @@
+module Inferno
+  module DSL
+    class JWKS
```

Comment:
Who are the intended users of this class? Test authors or core developers? There's no documentation, and the level of necessary documentation will vary depending on the audience.

Reply:
I think test authors. can be used on any test kit that uses jwt.
I'll add the documentation.